exploratory-tests/tesztterv.md

```diff
@@ -0,0 +1,126 @@
+Test plan
```

Comment:
Nincs feltültetve se az author, se a contribute-orok. Nem tudom akkor, hogy ki dolgozott ezen a feladaton. Kérlek add hozzá ezeket a fájl tetejére, valamint ahol releváns, ott is.

Reply:
Javítva.